UltraStar Play/Assets/Scenes/Options/RecordingOptions/AmplificationSlider.cs

```diff
@@ -12,7 +12,7 @@ public class AmplificationSlider : TextItemSlider<int>
     protected override void Awake()
     {
         base.Awake();
-        Items = new List<int> { 0, 6, 12, 18 };
+        Items = new List<int> { 0, 3, 6, 9, 12, 15, 18 };
```

Comment:
why did you add more options here? Don't we want to keep it as simple as possible? I don't think the additional steps will improve peoples gameplay because people are not machines and their distance from the microphones varies a bit while playing.

Reply:
When seeing the impact on the visualization, I just felt that 6 db was too much, but 0 was too little.